src/tests/multihost/ad/conftest.py

```diff
@@ -396,6 +397,26 @@ def restoresssdconf():
     request.addfinalizer(restoresssdconf)
 
 
+@pytest.fixture(scope="function")
+def range_retr_mods(session_multihost, fetch_ca_cert, request):
```

Comment:
Hi we are trying to avoid any direct calls to AD ldap, this should be replaced by powershell if possible.
If not possible I suggest to use the ldifde https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/cc731033(v=ws.11)  and other tools available directly on the AD.
This way we do not have the issues with ssl/tls on windows 2012.

Reply:
Modified the fixture to use the ldifde.exe